src/style.css

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: "Montserrat", sans-serif;
+  overflow: hidden;
+  margin: -21px 0 0 0;
+}
+
+.container {
+  display: flex;
+}
+
+.bg-one {
+  background-color: #6befa3;
+  height: 105vh;
+  width: 40vw;
+  position: relative;
+  z-index: 0;
+  overflow: hidden;
+}
+
+.bg-one::before {
+  content: "";
+  position: absolute;
+  background-color: #efefef;
+  height: 150%;
+  width: 100%;
+  right: 0;
+  margin-right: calc(-150% / 2);
+  top: 50%;
+  margin-top: calc(-1450px / 2);
+  border-radius: 50% 0% 0% 50%;
+  z-index: -1;
+}
+
+.bg-two {
+  content: "";
+  height: 105vh;
+  width: 60vw;
+  background-color: #efefef;
+}
+
+.footer {
+  bottom: 97px;
+  right: 188px;
+  position: fixed;
+  text-align: center;
+}
+
+.footer p {
+  font-size: 16px;
+  color: black;
+  font-weight: 400;
+}
+
+@media (min-device-width: 0px) and (max-device-width: 750px) {
+  .container {
+    flex-flow: column;
+  }
+
+  .bg-one {
+    background-color: #6befa3;
+    height: 60vh;
+    width: 100vw;
+    position: relative;
+    z-index: 0;
+    overflow-x: hidden;
+  }
+
+  .bg-one::before {
+    content: "";
+    position: absolute;
+    background-color: #efefef;
+    height: 30%;
+    width: 150%;
+    top: 100%;
+    left: 50%;
+    right: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
You could just use transform scale to stretch the circle instead of changing the width and adjusting the position like this, would save only one line of code anyway, I guess it's not a big deal.

Reply:
I don't know why but scale just isn't working